### PR TITLE
Fix code in exercises

### PR DESCRIPTION
**Pull request by lagru**
Addresses a few errors when executing the notebooks. I also took the liberty to use a more sensible and consistent numbering of parts and files.

### CODE REVIEW

**Comment by lagru** [comment]
Ready to merge. The reported failures when building
```
tutorial/03_segmentation_and_measuring.md: WARNING: Executing notebook failed: CellExecutionError [mystnb.exec]
tutorial/03_segmentation_and_measuring.md: WARNING: Notebook exception traceback saved in: tutorial/_build/html/reports/03_segmentation_and_measuring.err.log [mystnb.exec]
tutorial/03_segmentation_and_measuring.md:23: ERROR: Document orsection may not begin with a transition
```
are due to missing points that are intended to be picked by hand.

**Comment by JoOkuma** [comment]
today is a holiday here, I'll take a look at it tomorrow, @lagru

File: tutorial/index.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ unstuck!
 ## Instructors
 
 The instructors for this workshop are
-- Jordão Bragantini
-- Erick Martins Ratamero
 - Stefan van der Walt
```
Comment (stefanv) [inline tutorial/index.md]: Swap Lars and Stéfan, so Lars appears first.

**Review by stefanv — APPROVED** [review]

File: tutorial/index.md (hunk 1 above)
Comment (lagru) [reply tutorial/index.md]: I thought to list them in speaking order. But I don't mind switching them around.

**Comment by JoOkuma** [comment]
@lagru, the jupyters worked and everything seems correct, except it took me a minute or two to be able to load the `.md` as a jupyter notebook.
I didn't know how that worked and I had to `right click` it and select the `Notebook` option. Otherwise it was loaded as plain markdown. 

It may be worth updating the "Running the notebooks" section on `install_dependencies.md`

**Comment by lagru** [comment]
Thanks @JoOkuma. That's a good point about needing to right-click in jupyter lab. In think that's a limitation of jupytext. Jupyter notebook opens these fine. Would you like to add a comment? :)

**Pull request merged**